data/campaigns/Eastern_Invasion/utils/abilities.cfg

```
@@ -527,3 +527,14 @@ When an enemy unit of lower level within a 2 hex radius engages in combat, its a
         apply_to=fire,arcane
     [/resistance]
 #enddef
+
+#define ABILITY_ARCANE
+    [damage_type]
+        id=arcane_damage_blessing
+        name= _ "arcane blessing"
+        description= _ "All attacks combines the arcane type with the type of weapon used so that resistance to the arcane type does not penalize the user."
+        special_note=_ "This unit can use the arcane type when the opponent is particularly sensitive to it in relation to the weapon on which it is applied."
```

```suggestion
        description= _ "All attacks combine the arcane type with the type of the weapon used, so that resistance to arcane does not penalize the user."
        special_note=_ "This unit's weapons are treated as arcane instead of the declared damage type if that would increase the damage."
```

I think the description has room for improvement, but at least that fixes the grammar. I thought the wording of your special note was kinda confusing so I completely reworded it for you, but it too could maybe use some further work.